Makefile

```diff
@@ -693,7 +693,7 @@ TARGETCONFIG_FLAGS = -add-include '"evercrypt_targetconfig.h"'
 # corresponding feature, one of these variables needs to be overridden.
 E_HASH_BUNDLE=-bundle EverCrypt.Hash+EverCrypt.Hash.Incremental=[rename=EverCrypt_Hash]
 MERKLE_BUNDLE=-bundle 'MerkleTree+MerkleTree.EverCrypt+MerkleTree.Low+MerkleTree.Low.Serialization+MerkleTree.Low.Hashfunctions=MerkleTree.*[rename=MerkleTree]'
-CTR_BUNDLE=-bundle EverCrypt.CTR=EverCrypt.CTR.*
+CTR_BUNDLE=-bundle EverCrypt.CTR.*
```

Comment:
why do you need to keep this bundle?

Reply:
yes, there is still EverCrypt.CTR.Keys, which contains the key-expansion wrappers shared between AEAD and CTR... happy to do a followup to rename this module?